dbms/src/Flash/Planner/Plans/PhysicalTopN.cpp

```diff
@@ -79,7 +79,7 @@ void PhysicalTopN::buildPipelineExecGroupImpl(
     // TODO find a suitable threshold is necessary; 10000 is just a value picked without much consideration.
     if (group_builder.concurrency() * limit <= 10000)
     {
-        executeLocalSort(exec_context, group_builder, order_descr, limit, context, log);
+        executeLocalSort(exec_context, group_builder, order_descr, limit, false, context, log);
```

Comment:
It looks like this is similar to fine grained.
How about using the same spilling strategy?

Reply:
But for non fine grained case, one operator spill usually will not block others, that is why I only handle it for fine grained executor.

Reply:
But topN is usually together with sender, so there are similar situations.
I think it's all right.